modules/module/BaseImageMaskModel.py

```diff
@@ -114,12 +114,21 @@ def save_mask(self):
 
 class BaseImageMaskModel(metaclass=ABCMeta):
     @staticmethod
-    def __get_sample_filenames(sample_dir: str) -> [str]:
+    def __get_sample_filenames(sample_dir: str, include_subdirectories: bool = False) -> [str]:
+        def __is_supported_image_extension(filename: str) -> bool:
+            ext = os.path.splitext(filename)[1]
+            return ext in path_util.supported_image_extensions() and '-masklabel.png' not in filename
```

Comment:
I was not 100% sure so left not touched but ` path_util.is_supported_image_extension(ext) and '-masklabel.png' not in filename`  might be same and better.